### ISSUE TITLE
"renew" command fails in normal use without useful error

### ISSUE THREAD

**Issue by donmccurdy** [certbot/certbot#3824]
Hi,

I created my certificate with something along these lines:

```bash
./certbot-auto certonly -n -d www.domain1.com -d www.domain2.com --webroot --webroot-path /home/donmccurdy/public --expand
```

Later, to renew my certificate manually, and tried this:

```bash
./certbot-auto renew
```

Output:

```
➜  ~ ./certbot-auto renew 
Requesting root privileges to run certbot...
  /home/donmccurdy/.local/share/letsencrypt/bin/letsencrypt renew
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Currently, the renew verb is only capable of renewing all installed certificates that are due to
be renewed; individual domains cannot be specified with this action. If you would like to renew
specific certificates, use the certonly command. The renew verb may provide other options for
selecting certificates to renew in the future.
```

I couldn't parse that error, and the log wasn't any more specific, so for the last few months I've been manually creating a new certificate instead of renewing the existing one. I finally tracked the problem down, [thanks to this thread](https://community.letsencrypt.org/t/cert-renewal-error/11278). An entry had appeared in `cli.ini` (a file I never edited myself, that I'm aware of) hardcoding those domains, and the `renew` command apparently refers to that file.

Because domains were not specified in the command I issued, could this error message be more explicit about why it is failing? Or perhaps call some attention to the setting it inherits from `cli.ini`?

Cheers

**Comment by pde** [comment]
You probably have your domains listed in an `/etc/letsencrypt/cli.ini` file. Unfortunately, the `renew` subcommand is not yet compatible with that usage pattern (see #2305 to track progress on fixing this).  You could remove the domains from the `cli.ini` file, or perform renewal using the `certonly` sub-command instead.

See: https://certbot.eff.org/docs/using.html#renewal for some more docs on this.

**Comment by pde** [comment]
@ohemorange this is an example of why we should fix #2305 :)

**Comment by donmccurdy** [comment]
Hi @pde — unfortunately I think there is also a usability issue here, and unaddressed, the fix you describe will make it worse. I did have domains listed in `cli.ini`, but I didn't know they were there. Perhaps I followed some other usage pattern when trying to set up LetsEncrypt last year, and I've forgotten. In any case, they were not the domains I wanted to renew. When I created the current certificate, I used `certbot certonly -d`.

My problem is not that `renew` didn't support the domain options in my `cli.ini`. If it had silently parsed `cli.ini` and renewed for *those* domains, rather than the domains in my actual current certificate, it would have broken my site.

The problem, from my perspective, is that a config file I didn't know about was breaking my `renew` command, and the error message did not provide enough information for me to debug this. I've been re-creating my certificates manually for months, as a workaround. Other users have mentioned the same, in the thread linked above:

> it's not at all clear that letsencrypt-auto renew decides to read all the options in cli.ini before it does its work. I'm super stoked on the simplicity of the command in general, but it took a significant amount of time for me to figure out where the hell it was getting specific domains from, or why it was telling me agree-dev-preview was deprecated when I wasn't typing it.

Either of two things would improve usability here:

1. If `cli.ini` may affect the current subcommand, log some message like `Loading configuration from /etc/letsencrypt/cli.ini`.
2. If a subcommand fails because of incorrect usage, like this, consider mentioning `cli.ini` in the error message.

Thanks!

(Aside from the usability concern, of course, I would love to see #2305 implemented).
